### PR TITLE
Added docker detatched option

### PR DESCRIPTION
**Pull request by anandhu-eng**

### CODE REVIEW

File: .github/workflows/test-mlperf-inference-abtf-poc.yml [hunk 1]
```diff
@@ -34,6 +35,7 @@ jobs:
           python-version: "3.8"
         include:
           - os: windows-latest
+            docker: ""
```
Comment (arjunsuresh) [inline .github/workflows/test-mlperf-inference-abtf-poc.yml:38]: cant we install docker on windows non interactively?

Comment (anandhu-eng) [reply .github/workflows/test-mlperf-inference-abtf-poc.yml]: @arjunsuresh , there is command line installation method for windows in official docker [website](https://docs.docker.com/desktop/install/windows-install/). Mybe with some workaround its possible?

Comment (arjunsuresh) [reply .github/workflows/test-mlperf-inference-abtf-poc.yml]: It works but only for Windows docker images which CM does not support currently.

File: .github/workflows/test-mlperf-inference-abtf-poc.yml [hunk 2]
```diff
@@ -47,6 +52,61 @@ jobs:
...
+    - name: Install Docker on macos
+      if: runner.os == 'macOS-deactivated'
+      run: |
+        brew update
+        brew install --cask docker
+
+    - name: Start Docker Daemon on macos
+      if: runner.os == 'macOS-deactivated'
+      run: |
+        open /Applications/Docker.app
```
Comment (anandhu-eng) [inline .github/workflows/test-mlperf-inference-abtf-poc.yml:65]: When the docker application starts up for the first time, it asks for accepting the aggreements then only proceeds to starting the docker daemon. Maybe we have to find some better way?
Screenshot from 2024-11-06 18-52-23

**Review by arjunsuresh — APPROVED** [review]

**Pull request merged**